src/components/DocumentationTopic.vue

```diff
@@ -647,6 +676,47 @@ export default {
 <style scoped lang="scss">
 @import 'docc-render/styles/_core.scss';
 
+$space-size: 15px;
+
+.declaration-list-menu {
+  position: relative;
+  margin: 0 !important;
```

Comment:
It would be awesome if we could clean this up to not require `!important`.

Reply:
Actually I don't even think setting margin here is needed. I removed it.